proposals/issue_triage.md

```diff
@@ -49,6 +49,16 @@ The following are necessary and sufficient conditions for an issue to be conside
 
   * The oncall will attempt to satisfy the above criterion or reassign to an appropriate WG if there is some question
 
+## Triage guideline
```

Comment:
How about adding some of the guidelines we discussed on the email thread for figuring out how to classify issues as bug vs. feature and the appropriate priority.

Also can you add a section about how to get the appropriate permissions? i.e. filing a PR to gain access to the github group project maintainers?